handler/commentHandler.go

```diff
@@ -35,6 +35,8 @@ const (
 	removeMilestoneConstant  string = "RemoveMilestone"
 	assignReviewerConstant   string = "AssignReviewer"
 	unassignReviewerConstant string = "UnassignReviewer"
+	messageConstant          string = "message"
```

Comment:
We shouldn't need an alias here.  The substitution of the alias can be achieved in `parse`.  Take a look at how `setTitle` is implemented with the edit alias.

Reply:
Done 👍